tools/min-base-version

```diff
@@ -28,14 +28,14 @@ from glob import glob
 proj_dir = os.path.dirname(os.path.dirname(os.path.realpath(sys.argv[0])))
 
 max_version = '0'
-for manifest in glob(os.path.join(proj_dir, 'pkg/*/manifest.json')):
+for manifest in glob(os.path.join(proj_dir, 'pkg/*/manifest.json.in')):
```

Comment:
This is probably a follow up but we cannot conflate all the "requires" here. Certain distros don't ship all the packages at the same time. We need to start doing these individually or manually curating these. Such a change could be a follow up.